continuous-testing/README.md

```diff
@@ -10,7 +10,7 @@ In this section, we provide information about potentially useful *use cases* for
 
 ## Trade Studies
 
-Here we provide a table of trade offs, feature comparisons, and metrics for a given set of reference architectures mapped to use case tool solutions. Choice of comparison points are solicited directly from community members feedback.
+Here we provide a table of trade offs, feature comparisons, and metrics for a given set of reference architectures mapped to use case tool solutions. Choice of comparison points are solicited directly from community member feedback.
```

Comment:
Nice catch! I'm making similar changes to the Documentation guide to reflect.

Reply:
Thanks